sphinx/templates/latex/sphinxmessage.sty_t

```diff
@@ -3,3 +3,11 @@
 \renewcommand{\sphinxnonalphabeticalgroupname}{<%= _('Non-alphabetical') | e %>}
 \renewcommand{\sphinxsymbolsname}{<%= _('Symbols') | e %>}
 \renewcommand{\sphinxnumbersname}{<%= _('Numbers') | e %>}
+\def\pageautorefname{<%= _('page') | e %>}
+
+<%= addtocaptions %>{\renewcommand{\figurename}{<%= figurename[0].strip() | e %>}}
```

Comment:
I use simple string-concatenation to build `\addto\captions...` macro. As side effect, meaningless braces will be generated to non-multilingual document (mainly Japanese docs).